components/ILIAS/Contact/PRIVACY.md

```diff
@@ -0,0 +1,74 @@
+- Furthermore, when a user profiles is published, it gives all users access to the profile via the "Who is online" tool
+  and, through that, the option to request contact with that profile.
+  - This can be activated in your profile under "Profile and Privacy > Publish Profile > Visible for logged-in Users"
+    setting.
+  - This setting does not allow or prevent the contact request itself.
+- A contact can be used for two actions:
+  - You can add contacts to your mailing lists
+  - You can invite contacts to chatrooms
+- Furthermore, approved contacts are shown within the "Who is online?" tool
+- The component is not aware of the data being passed through mails or chats. It does not know anything about
+  the origin (i.e. course) of the content or whether it contains user related data or not.
+
+## Data being stored
+
+- Each contact request is stored, including the ID of the requesting user and the ID of the requested user, as well as a
+  timestamp of the request time. This does not require the consent of both target users.
+- When the requested user actively ignores the request, this data entry is added with an "ignore" flag.
+- After confirmation, each contact is stored, including the ID of the requesting user and the ID of the requested user as
+  well as a timestamp of the request time.
+- The storage of requests and approved contacts are separated.
+- Each contact added as an entry to a mailing list is stored by using the user_id.
+- Any user relations that originate from the contact is not related or depending on the active contact. There is no
+  traceable relation of contacts withing other components.
+
+## Data being presented
+
+- Contact data is presented within "Communication > Contacts" and within a separate section of the "Who is online?" tool.
+- The presentation includes a presentation of the user's profile (restricted depending on the user's profile publishing
+  settings) and control elements presenting the user's options for contact requests ("Confirm Contact"/"Ignore Contact").
+- Initially the options are restricted to requesting a contact.
+- The requesting user can cancel the request.
+- The requested user has the option to actively ignore or confirm the request.
+- Both contacts can unlink the contact without any further consent.
+- Each party can detect a change of the contact relation within its own contact options. The only exception to this
+  is the active ignoring of a request. The requesting user can not detect if his request was ignored.
+- A list of all contacts is shown as a selection when a user adds members to a newly created mailing list.
+- A contact relation cannot be perceived within the user interface by any user other than the contacts themselves.
+
+## Data being deleted
+
+- If a contact request is canceled, the stored entry is removed without any traceable footprint.
+- If a contact request is ignored, the flagged entry in the database will be preserved to prevent another request.
```

Comment:
`to prevent another request.` -> maybe better?: `to prevent a repeated contact request from the same initiator`